src/Components/Users/Users.tsx

```diff
@@ -1,7 +1,172 @@
-import React from 'react'
+import { Fragment, useContext, useEffect, useState } from 'react'
+import styles from './Users.module.css'
+import { HiDotsVertical } from "react-icons/hi";
```

Comment:
You change the name of the icons as you like as follows
`import { HiDotsVertical as ThreeDots } from "react-icons/hi";`

This will enable you to give meaningful names to the icons.

Reply:
It is also not recommended to use multiple icon sets as every icon set will have its own style and you'll end up with different icon styles which is not good from a design perspective.

You can depend for one such as Feather icon set, and if the icons that you need are not in it you can look for alternative package to select from. This will reduce the possibility of having various icon sets with different styles.